DoctrineMigrationsBundle.php

```diff
@@ -14,7 +14,7 @@
  */
 class DoctrineMigrationsBundle extends Bundle
 {
-    public function build(ContainerBuilder $container)
+    public function build(ContainerBuilder $container): void
```

Comment:
The class is final, so this is technically a breaking change. You should use a phpdoc annotation instead:

```suggestion
    /** @return void */
    public function build(ContainerBuilder $container)
```

Reply:
I also suggest adding `@final since 3.3.0` (or `3.4.0` depending on what is the next minor version) on that class

Reply:
The next minor version should be 3.3.0.